falcon/response.py

```diff
@@ -425,6 +425,9 @@ def set_cookie(self, name, value, expires=None, max_age=None,
         if http_only:
             self._cookies[name]['httponly'] = http_only
 
+        if same_site:
+            self._cookies[name]['samesite'] = same_site[0]
```

Comment:
Why did you choose to make `same_site` a tuple instead of just a string?

Reply:
Yes, it's my mistake. We can pass it as a string, but the main question is - do we need to raise an error if a string doesn't equal to `Lax` or `Strict`?

Reply:
> do we need to raise an error if a string doesn't equal to Lax or Strict?

No. Note that no other parameters in this method are validated.

Reply:
I second @RonRothman on the parameter type, a string parameter sounds easier for the API users unless there are use cases where a list might be useful?

I'm admittedly a bit divided on the validation question. @kgriffs  thoughts?
I would say we do actually validate some parameters, for instance, `max_age` is cast to `int` etc. Now that we are on Python 3, defining an `enum` with supported values is another option to consider. OTOH we are not using `enum`s anywhere else in Falcon (at least not yet).
NB if we decide to validate anything, we could also consider supporting the experimental `'None'` value too.